src/resources/ui/preferences.ui

```diff
@@ -145,4 +146,4 @@
         <property name="step-increment">1</property>
     </object>
 
-</interface>
\ No newline at end of file
+</interface>
```

Comment:
Please, remove new line here

Reply:
The newline seems to be automatically added after I remove it.

Reply:
Hi. I just wanted to ask what the state of this PR is. Are planing on merging it soon?

Reply:
> Hi. I just wanted to ask what the state of this PR is. Are planing on merging it soon?

Nope, because PR is not ready to merge (All checks have failed)

Reply:
> > Hi. I just wanted to ask what the state of this PR is. Are planing on merging it soon?
> 
> Nope, because PR is not ready to merge (All checks have failed)

Ok, all checks have passed and this PR is now ready to be merged.

> > Hi. I just wanted to ask what the state of this PR is. Are planing on merging it soon?
> 
> Nope, because PR is not ready to merge (All checks have failed)

@win0err 
All checks have passed now and this PR is ready to be merged.